action_paymails.go

```diff
@@ -101,6 +102,12 @@ func (c *Client) NewPaymailAddress(ctx context.Context, xPubKey, address, public
 		return nil, err
 	}
 
+	// Check if the paymail address already exists
+	paymail, _ := getPaymailAddress(ctx, address, opts...)
```

Comment:
@pawellewandowski98 please check the error (just in case!)

Reply:
Done, thanks!